packages/cbioportal-utils/src/mutation/MutationAnnotator.ts

```diff
@@ -337,3 +342,11 @@ export function genomicLocationStringFromVariantAnnotation(
         variantAllele,
     });
 }
+
+export function getFusionMutations(mutations: Mutation[]) {
+    return mutations.filter(
+        (m: Mutation) =>
+            m.mutationType &&
+            new RegExp('fusion', 'i').test(m.mutationType.toLowerCase())
```

Comment:
you don't need to lowercase mutationType since the regex is case insensitive.      Also, this regexp only needs to be instantiated once.  Making a new one inside loop is a performance drain.  Since there might well be tens of thousands of mutations, it's worth optimizing.  Just make the regexp outside the function.